### DISCUSSION TITLE
개인 일정을 적어주세요

### DISCUSSION THREAD

**Discussion by 42kko** [42webserv/42webserv#4] [Announcements]
개인 일정을 적어주세요

**Comment by Y2IN** [comment]
4/13 목요일 저녁 개인 사정
4/19 수요일 개인 사정
4/27 3시~4시 이후부터 가능

**Comment by chanwoong1** [comment]
4/12 수요일 안과

**Comment by gtd9511** [comment]
4/17 월요일 가스 점검 (지각)
4/19 수요일 개인 사정 (불참)
5/8   수요일 치과(지각)

**Comment by get6** [comment]
4/19 오후 2-4시 교육
4/21 오후 2-4시 교육
